test/test-msbuild.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#
+# Copyright (c) 2024 Sergey Kvachonok
+#
+# Permission to use, copy, modify, and/or distribute this software for any
+# purpose with or without fee is hereby granted, provided that the above
+# copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+# WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+# MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+# ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+# WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+# ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+# OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+. "${0%/*}/test.sh"
+
+for config in Debug Release; do
+    for useenv in true false; do
+        EXEC "" ${BIN}msbuild /p:UseEnv=${useenv} /p:Configuration=${config} "${TESTS}HelloWorld.vcxproj"
+        rm -rf "${TESTS}/${config}"
```

Comment:
Can we set `IntDir` and `OutDir` to `${CWD}` such that `msbuild` doesn't populate things to source directory?

Reply:
Good idea! I'll add a new commit addressing this.

Reply:
Resolved in https://github.com/mstorsjo/msvc-wine/pull/103/commits/345b115b8ee931ec6df457b4cd251412874cb71c.